app/views/assets/_asset.html.erb

```diff
@@ -1,8 +1,31 @@
-<% gallery_view_id = nil unless defined? gallery_view_id %>
+<%
+unless defined? gallery_view_id
+  gallery_view_id = nil
+end
+
+unless defined? deletable
```

Comment:
Isn't `deletable` always undefined? I didn't find a place, where you would be passing `deletable` local to partial.

Reply:
Mostly for future, for the ability to enable delete option in the context menu,  currently by enabled to steps only.